README.md

```diff
@@ -29,7 +29,7 @@ go test -tags examples -count 1 -v ./examples/users_example_test.go
 ```
 
```

Comment:
> Note that they are placed in Go test files (e.g. _users_example_test.go_) to be easily runnable.

This sentence involves only Admin APIs. I think we should transform synthetics and cloud export examples to be test files for consistency. This PR is a opportunity to do so.

Demos can be left unmodified.